src/conn/conn_api.c

```diff
@@ -1450,9 +1450,20 @@ __conn_rollback_to_stable(WT_CONNECTION *wt_conn, const char *config)
     WT_CONNECTION_IMPL *conn;
```

Comment:
(Specific to this file, not this line) per my comment above about `session_max`, I think you should modify `conn_session_size` in this file to add in the configured threads to that it doesn't come out of the user's number. There are already adjustments in that function for LSM threads, eviction, etc. This change belongs with them.